src/MeshVolume.js

```diff
@@ -70,15 +75,20 @@ export default class MeshVolume {
     this.meshRoot.position.copy(vec3xyz);
   }
 
-  setRotation(quaternion) {
-    this.meshPivot.rotation.copy(quaternion);
+  setRotation(eulerXYZ) {
+    this.meshPivot.rotation.copy(eulerXYZ);
+    this.updateClipFromBounds();
   }
 
   setResolution(x, y) {}
 
   setOrthoThickness(value) {}
 
-  setAxisClip(axis, minval, maxval, isOrthoAxis) {}
```

Comment:
was this a placeholder you had written as a TODO?

Reply:
yes, i had left it here as an empty function just in case some day it would need to actually do something, like the ones above it.  The raymarch and pathtrace representations have the same api, so it was here for consistency.   Some day there will be a typescript conversion and it will be declared as an interface.